Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -239,6 +239,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Bugzilla59580.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Effects\AttachedStateEffect.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Effects\AttachedStateEffectList.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)GitHub1702.cs" />
```

Comment:
Rename to Issue1702.cs

Reply:
I noticed that in `Issue####` classes some other `github` number is specified and different description.
For example `Issue1461.cs` created almost 2 years ago
https://github.com/xamarin/Xamarin.Forms/blob/b794caf7be33e5d3bf8ea39ba719cdbfbc008315/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1461.cs#L30-L31
And `Github1461.cs` created a couple of months ago
https://github.com/xamarin/Xamarin.Forms/blob/b794caf7be33e5d3bf8ea39ba719cdbfbc008315/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Github1461.cs#L13-L14
`Github1461.cs` is correct for this repository https://github.com/xamarin/Xamarin.Forms/issues/1461

I was confused. Is renaming to `Issue####` still valid?

Reply:
Yea :-/ Right now there's a mix of numbers that relate to the old github repo before the project went public.  For now we're still just creating the Issues as Issue#### when there isn't a number collision